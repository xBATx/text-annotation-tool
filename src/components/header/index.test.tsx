import React from 'react';
import renderer from 'react-test-renderer';
import Header from './index';

it('renders without crashing', () => {
  const wrapper = renderer
    .create(<Header title="Hello title">Hello I'm child</Header>)
    .toJSON();
  expect(wrapper).toMatchSnapshot();
});
