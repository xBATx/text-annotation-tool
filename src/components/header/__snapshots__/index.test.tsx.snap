// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
Array [
  <div
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "blue",
        "display": "flex",
        "height": 60,
        "justifyContent": "flex-start",
        "position": "relative",
        "width": "100%",
        "zIndex": 4,
      }
    }
  >
    <div
      style={
        Object {
          "color": "white",
          "left": "50%",
          "position": "absolute",
        }
      }
    >
      Hello title
    </div>
  </div>,
  "Hello I'm child",
]
`;
